Task20/ClassLibrary3/Tests.cs

```diff
@@ -30,6 +31,12 @@ public void LoginUserTutBy()
             tutby.Login(username, password);
 
             var loginState = tutby.IsUserLogin();
+
+            ITakesScreenshot screenshotDriver = Driver as ITakesScreenshot;
+            Screenshot screenshot = screenshotDriver.GetScreenshot();
+            String fp = "D:\\" + "snapshot" + "_" + DateTime.Now.ToString("dd_MMMM_hh_mm_ss_tt") + ".png";
+            screenshot.SaveAsFile(fp, ScreenshotImageFormat.Png);
+
```

Comment:
It would be better to take this lines into separate method at the Base class level. It'll allow you to use this code it whole project.

Reply:
Changes are made in the new commit